apps/user/src/components/common/Header/Header.tsx

```diff
@@ -67,14 +69,16 @@ const Header = () => {
                 )}
             </HeaderBar>
             <NavigationBar>
-                {NAVIGATION_DATA.map(({ route, name }, index) => (
-                    <UnderLineButton
-                        key={`navigation ${index}`}
-                        active={route === pathName}
-                        onClick={() => router.push(route)}>
-                        {name}
-                    </UnderLineButton>
-                ))}
+                {NAVIGATION_DATA.map(({ route, name }, index) => {
+                    return (
+                        <UnderLineButton
+                            key={`navigation ${index}`}
+                            active={route === pathName.match(/^\/[^\/]*/s)?.[0]}
```

Comment:
```suggestion
                            active={route === pathName}
```

이렇게해도 되는걸로 알고있는데 저렇게 하신 이유가 있나여?

Reply:
공지사항 페이지에서 상세히 보기를 눌렀을 때 active가 false가 되길래 `/notice/1` 이라는 문자열을 `/notice` 로 변경하려고 했습니다. 그런데 그때 제가 살짝 미쳐가지고 지금 보니 많이 이상해 보이네요. 차라리 뺄까요?

Reply:
/notice/1 해당 이슈는 저도 알고있었던 이슈긴한ㄷ 더 좋은 방법을 한번 같이 찾아봐요~ !

Reply:
넵!